diff --git a/src/components/navbar/RightNav.js b/src/components/navbar/RightNav.js
index a8846dc..fde78f1 100644
--- a/src/components/navbar/RightNav.js
+++ b/src/components/navbar/RightNav.js
@@ -128,13 +128,13 @@ const RightNav = ({ open }) => {
     <Ul open={open}>
       <li><a href="/">Home</a></li>
       <li><a href="/about">About Us</a></li>
-      <li id="dropdown-button"><a href="/teamintro">Team</a>
+      <li id="dropdown-button">Team
         <ul id="dropdown-menu">
-          <li id="firstitem"><a href="/teamintro">Team Intro</a></li>
-          <li><a href="/team2018">Team 2018-2019</a></li>
-          <li><a href="/team2019">Team 2019-2020</a></li>
+          <li><a href="/teamintro">Team Intro</a></li>
+          <li><a href="/team2021">Team 2021-2022</a></li>
           <li><a href="/team2020">Team 2020-2021</a></li>
-          <li id="lastitem"><a href="/team2021">Team 2021-2022</a></li>		  
+          <li><a href="/team2019">Team 2019-2020</a></li>
+          <li><a href="/team2018">Team 2018-2019</a></li>
         </ul>
       </li>
       <li><a href="/sponsors">Sponsors</a></li>
diff --git a/src/components/sponsors/sponsor.js b/src/components/sponsors/sponsor.js
new file mode 100644
index 0000000..56c7fad
--- /dev/null
+++ b/src/components/sponsors/sponsor.js
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from 'react';
+import './sponsors_body.css';
+
+const OVERLAY_STYLES = {
+    position: 'fixed',
+    top: 0,
+    left: 0,
+    right: 0,
+    bottom: 0,
+    backgroundColor: 'rgba(0, 0, 0, .7)',
+    zIndex: 1000
+}
+
+function SponsorPopup({ open, onClose, title: Title, description: Description, url: URL }) {
+    if (!open) return null
+    return (
+        <div style={OVERLAY_STYLES}>
+            <div className="sponsorPopUp">
+                <button onClick={onClose}>&#x2715;</button>
+                <p className="Title">{Title}</p>
+                <p>{Description}</p>
+                <a href={URL}>{URL}</a>
+            </div>
+        </div>
+    )
+}
+
+export default function SponsorModal({ img: Img, title: Title, description: Description, url: URL }) {
+    const [isOpen, setisOpen] = useState(false)
+    const [isShown, setIsShown] = useState(false);
+    return (
+        <div>
+            <button
+                className={isShown ? "sponsorHover" : "sponsor"}
+                onClick={() => setisOpen(true)}
+                onMouseEnter={() => setIsShown(true)}
+                onMouseLeave={() => setIsShown(false)}>
+                <div>
+                    <img src={Img} />
+                    {isShown && (
+                        <div>Click to learn more!</div>
+                    )}
+                </div>
+            </button>
+            <SponsorPopup
+                open={isOpen}
+                onClose={() => setisOpen(false)}
+                title={Title}
+                description={Description}
+                url={URL}
+            />
+        </div>
+    )
+}
\ No newline at end of file
diff --git a/src/components/sponsors/sponsorEight.js b/src/components/sponsors/sponsorEight.js
deleted file mode 100644
index 6d22820..0000000
--- a/src/components/sponsors/sponsorEight.js
+++ /dev/null
@@ -1,29 +0,0 @@
-import React, {useState} from 'react'
-import './sponsors_body.css';
-
-const OVERLAY_STYLES = {
-    position: 'fixed',
-    top: 0,
-    left: 0,
-    right: 0,
-    bottom: 0,
-    backgroundColor: 'rgba(0, 0, 0, .7)',
-    zIndex: 1000
-}
-
-export default function SponsorEight({open, children, onClose}){
-    if (!open) return null
-    return (
-        <div style = {OVERLAY_STYLES}>
-	        <div className="sponsorPopUp">
-				<button onClick={onClose}>&#x2715;</button>
-				<p className="Title">Fluid Flow</p>
-				<p>FluidFlow is a pipe flow design software allowing 
-                users to design pressure drop systems of any needs. 
-                Flows can include non-newtonian and slurry flow, 
-                two-phase flow, thermal energy transfer and compressible flow for example.</p>
-                <a href="https://fluidflowinfo.com">https://fluidflowinfo.com</a>
-            </div>
-        </div>
-    )
-}
\ No newline at end of file
diff --git a/src/components/sponsors/sponsorEighteen.js b/src/components/sponsors/sponsorEighteen.js
deleted file mode 100644
index c24a61b..0000000
--- a/src/components/sponsors/sponsorEighteen.js
+++ /dev/null
@@ -1,30 +0,0 @@
-import React, {useState} from 'react'
-import './sponsors_body.css';
-
-const OVERLAY_STYLES = {
-    position: 'fixed',
-    top: 0,
-    left: 0,
-    right: 0,
-    bottom: 0,
-    backgroundColor: 'rgba(0, 0, 0, .7)',
-    zIndex: 1000
-}
-
-export default function SponsorEighteen({open, children, onClose}){
-    if (!open) return null
-    return (
-        <div style = {OVERLAY_STYLES}>
-	        <div className="sponsorPopUp">
-				<button onClick={onClose}>&#x2715;</button>
-				<p className="Title">Second Order Effects</p>
-				<p>Develops and designs many solutions to challenges that 
-                are solved by their broad range of expertise in electronics, 
-                automotive, healthcare, and industrial automation industries. 
-                Reaching a solution that produces maximized results.
-                </p>
-                <a href="https://soeffects.com/">https://soeffects.com/</a>
-			</div>
-        </div>
-    )
-}
\ No newline at end of file
diff --git a/src/components/sponsors/sponsorEleven.js b/src/components/sponsors/sponsorEleven.js
deleted file mode 100644
index c7853f2..0000000
--- a/src/components/sponsors/sponsorEleven.js
+++ /dev/null
@@ -1,28 +0,0 @@
-import React, {useState} from 'react'
-import './sponsors_body.css';
-
-const OVERLAY_STYLES = {
-    position: 'fixed',
-    top: 0,
-    left: 0,
-    right: 0,
-    bottom: 0,
-    backgroundColor: 'rgba(0, 0, 0, .7)',
-    zIndex: 1000
-}
-
-export default function SponsorEleven({open, children, onClose}){
-    if (!open) return null
-    return (
-        <div style = {OVERLAY_STYLES}>
-	        <div className="sponsorPopUp">
-				<button onClick={onClose}>&#x2715;</button>
-				<p className="Title">Mechanical and Aerospace Engineering Department</p>
-				<p>Department offering comprehensive resources for students, 
-                faculty, and staff at UC Davis (offering undergraduate advising and research opportunities).
-                </p>
-                <a href="https://mae.ucdavis.edu/">https://mae.ucdavis.edu/</a>
-			</div>
-        </div>
-    )
-}
\ No newline at end of file
diff --git a/src/components/sponsors/sponsorFifteen.js b/src/components/sponsors/sponsorFifteen.js
deleted file mode 100644
index 8970ed6..0000000
--- a/src/components/sponsors/sponsorFifteen.js
+++ /dev/null
@@ -1,30 +0,0 @@
-import React, {useState} from 'react'
-import './sponsors_body.css';
-
-const OVERLAY_STYLES = {
-    position: 'fixed',
-    top: 0,
-    left: 0,
-    right: 0,
-    bottom: 0,
-    backgroundColor: 'rgba(0, 0, 0, .7)',
-    zIndex: 1000
-}
-
-export default function SponsorFifteen({open, children, onClose}){
-    if (!open) return null
-    return (
-        <div style = {OVERLAY_STYLES}>
-	        <div className="sponsorPopUp">
-				<button onClick={onClose}>&#x2715;</button>
-				<p className="Title">CITRIS</p>
-				<p>An institute opened by the university of california to help 
-                further the research might of the general university of california system, 
-                and help it to connect with the entrepreneurial spirit of silicon valley, 
-                bridging the gap between companies, students, and policy makers. 
-                </p>
-                <a href="https://citris-uc.org/">https://citris-uc.org/</a>
-			</div>
-        </div>
-    )
-}
\ No newline at end of file
diff --git a/src/components/sponsors/sponsorFive.js b/src/components/sponsors/sponsorFive.js
deleted file mode 100644
index a3267ae..0000000
--- a/src/components/sponsors/sponsorFive.js
+++ /dev/null
@@ -1,29 +0,0 @@
-import React, {useState} from 'react'
-import './sponsors_body.css';
-
-const OVERLAY_STYLES = {
-    position: 'fixed',
-    top: 0,
-    left: 0,
-    right: 0,
-    bottom: 0,
-    backgroundColor: 'rgba(0, 0, 0, .7)',
-    zIndex: 1000
-}
-
-export default function SponsorFive({open, children, onClose}){
-    if (!open) return null
-    return (
-        <div style = {OVERLAY_STYLES}>
-	        <div className="sponsorPopUp">
-				<button onClick={onClose}>&#x2715;</button>
-				<p className="Title">Seimens</p>
-				<p>Siemens is the largest manufacturing company in Europe. 
-                A multinational conglomerate company whose varied acquisitions 
-                allowed it to be involved in wind power, gas power, energy management, 
-                transportation, health care, automation and more.</p>
-                <a href="https://www.siemens.com/global/en.html">https://www.siemens.com/</a>
-			</div>
-        </div>
-    )
-}
\ No newline at end of file
diff --git a/src/components/sponsors/sponsorFour.js b/src/components/sponsors/sponsorFour.js
deleted file mode 100644
index e4f2ea7..0000000
--- a/src/components/sponsors/sponsorFour.js
+++ /dev/null
@@ -1,29 +0,0 @@
-import React, {useState} from 'react'
-import './sponsors_body.css';
-
-const OVERLAY_STYLES = {
-    position: 'fixed',
-    top: 0,
-    left: 0,
-    right: 0,
-    bottom: 0,
-    backgroundColor: 'rgba(0, 0, 0, .7)',
-    zIndex: 1000
-}
-
-export default function SponsorFour({open, children, onClose}){
-    if (!open) return null
-    return (
-        <div style = {OVERLAY_STYLES}>
-	        <div className="sponsorPopUp">
-				<button onClick={onClose}>&#x2715;</button>
-				<p className="Title">Valispace</p>
-				<p>Helps engineers streamline their engineering efforts by centralizing ‘loose’ engineering data into one place. 
-                This provides a way for engineers to maximize their time and effort and design with their system of detailed 
-                documentation and secured user deployment.
-                </p>
-                <a href="https://www.valispace.com/">https://www.valispace.com/</a>
-			</div>
-        </div>
-    )
-}
\ No newline at end of file
diff --git a/src/components/sponsors/sponsorFourteen.js b/src/components/sponsors/sponsorFourteen.js
deleted file mode 100644
index 8e34ef0..0000000
--- a/src/components/sponsors/sponsorFourteen.js
+++ /dev/null
@@ -1,28 +0,0 @@
-import React, {useState} from 'react'
-import './sponsors_body.css';
-
-const OVERLAY_STYLES = {
-    position: 'fixed',
-    top: 0,
-    left: 0,
-    right: 0,
-    bottom: 0,
-    backgroundColor: 'rgba(0, 0, 0, .7)',
-    zIndex: 1000
-}
-
-export default function SponsorFourteen({open, children, onClose}){
-    if (!open) return null
-    return (
-        <div style = {OVERLAY_STYLES}>
-	        <div className="sponsorPopUp">
-				<button onClick={onClose}>&#x2715;</button>
-				<p className="Title">Rockwell Automation</p>
-				<p>A provider of both information technology and industrial automation. 
-                Many of their products are recognized across the globe for excellence.
-                </p>
-                <a href="https://www.rockwellautomation.com/en-us.html">https://www.rockwellautomation.com/en-us.html</a>
-			</div>
-        </div>
-    )
-}
\ No newline at end of file
diff --git a/src/components/sponsors/sponsorNine.js b/src/components/sponsors/sponsorNine.js
deleted file mode 100644
index 8834ddc..0000000
--- a/src/components/sponsors/sponsorNine.js
+++ /dev/null
@@ -1,29 +0,0 @@
-import React, {useState} from 'react'
-import './sponsors_body.css';
-
-const OVERLAY_STYLES = {
-    position: 'fixed',
-    top: 0,
-    left: 0,
-    right: 0,
-    bottom: 0,
-    backgroundColor: 'rgba(0, 0, 0, .7)',
-    zIndex: 1000
-}
-
-export default function SponsorNine({open, children, onClose}){
-    if (!open) return null
-    return (
-        <div style = {OVERLAY_STYLES}>
-	        <div className="sponsorPopUp">
-				<button onClick={onClose}>&#x2715;</button>
-				<p className="Title">Aqua Environment</p>
-				<p>Specialize in manufacturing high pressure components such as 
-                valves and regulators for gas and fluid control. Products are used 
-                in wide rage of commercial firefighting, military, sports-diving, 
-                natural gas, natural gas fill stations, and other applications (shipping worldwide).</p>
-                <a href="https://www.aquaenvironmentinc.com/index.html">https://www.aquaenvironmentinc.com/</a>
-			</div>
-        </div>
-    )
-}
\ No newline at end of file
diff --git a/src/components/sponsors/sponsorNineteen.js b/src/components/sponsors/sponsorNineteen.js
deleted file mode 100644
index bb4ed67..0000000
--- a/src/components/sponsors/sponsorNineteen.js
+++ /dev/null
@@ -1,29 +0,0 @@
-import React, {useState} from 'react'
-import './sponsors_body.css';
-
-const OVERLAY_STYLES = {
-    position: 'fixed',
-    top: 0,
-    left: 0,
-    right: 0,
-    bottom: 0,
-    backgroundColor: 'rgba(0, 0, 0, .7)',
-    zIndex: 1000
-}
-
-export default function SponsorNineteen({open, children, onClose}){
-    if (!open) return null
-    return (
-        <div style = {OVERLAY_STYLES}>
-	        <div className="sponsorPopUp">
-				<button onClick={onClose}>&#x2715;</button>
-				<p className="Title">Ubiquiti Networks</p>
-				<p>A provider of multi-application network operating systems 
-                that offers many services such as identification across networks, 
-                network management, and IT customization.
-                </p>
-                <a href="https://www.ui.com/">https://www.ui.com/</a>
-			</div>
-        </div>
-    )
-}
\ No newline at end of file
diff --git a/src/components/sponsors/sponsorOne.js b/src/components/sponsors/sponsorOne.js
deleted file mode 100644
index 07e349f..0000000
--- a/src/components/sponsors/sponsorOne.js
+++ /dev/null
@@ -1,28 +0,0 @@
-import React, {useState} from 'react'
-import './sponsors_body.css';
-
-const OVERLAY_STYLES = {
-    position: 'fixed',
-    top: 0,
-    left: 0,
-    right: 0,
-    bottom: 0,
-    backgroundColor: 'rgba(0, 0, 0, .7)',
-    zIndex: 1000
-}
-
-export default function SponsorOne({open, children, onClose}){
-    if (!open) return null
-    return (
-        <div style = {OVERLAY_STYLES}>
-	        <div className="sponsorPopUp">
-				<button onClick={onClose}>&#x2715;</button>
-				<p className="Title">ETEL</p>
-				<p>ETEL is a company based in Switzerland which designs and builds motion systems in-house. 
-                They create systems that meet a range of needs from high-precision to high-power. 
-                </p>
-                <a href="https://www.etel.ch">https://www.etel.ch</a>
-			</div>
-        </div>
-    )
-}
\ No newline at end of file
diff --git a/src/components/sponsors/sponsorSeven.js b/src/components/sponsors/sponsorSeven.js
deleted file mode 100644
index 90b4bb2..0000000
--- a/src/components/sponsors/sponsorSeven.js
+++ /dev/null
@@ -1,28 +0,0 @@
-import React, {useState} from 'react'
-import './sponsors_body.css';
-
-const OVERLAY_STYLES = {
-    position: 'fixed',
-    top: 0,
-    left: 0,
-    right: 0,
-    bottom: 0,
-    backgroundColor: 'rgba(0, 0, 0, .7)',
-    zIndex: 1000
-}
-
-export default function SponsorSeven({open, children, onClose}){
-    if (!open) return null
-    return (
-        <div style = {OVERLAY_STYLES}>
-	        <div className="sponsorPopUp">
-				<button onClick={onClose}>&#x2715;</button>
-				<p className="Title">AirCaster</p>
-                <p>AirCaster designs & builds air bearing systems that transport heavy loads 
-                across industries which includes but is not limited to aerospace, oil, defense, 
-                and automotive. They are known for their durability and customization to meet unique needs.</p>
-                <a href="https://www.aircaster.com">https://www.aircaster.com</a>
-            </div>
-        </div>
-    )
-}
\ No newline at end of file
diff --git a/src/components/sponsors/sponsorSeventeen.js b/src/components/sponsors/sponsorSeventeen.js
deleted file mode 100644
index e773d72..0000000
--- a/src/components/sponsors/sponsorSeventeen.js
+++ /dev/null
@@ -1,28 +0,0 @@
-import React, {useState} from 'react'
-import './sponsors_body.css';
-
-const OVERLAY_STYLES = {
-    position: 'fixed',
-    top: 0,
-    left: 0,
-    right: 0,
-    bottom: 0,
-    backgroundColor: 'rgba(0, 0, 0, .7)',
-    zIndex: 1000
-}
-
-export default function SponsorSeventeen({open, children, onClose}){
-    if (!open) return null
-    return (
-        <div style = {OVERLAY_STYLES}>
-	        <div className="sponsorPopUp">
-				<button onClick={onClose}>&#x2715;</button>
-				<p className="Title">Keysight Technology</p>
-				<p>Provides many services in technology consultation, design, 
-                and software to address pressing issues for business and technological advancement.
-                </p>
-                <a href="https://www.keysight.com/us/en/home.html">https://www.keysight.com/us/en/home.html</a>
-			</div>
-        </div>
-    )
-}
\ No newline at end of file
diff --git a/src/components/sponsors/sponsorSix.js b/src/components/sponsors/sponsorSix.js
deleted file mode 100644
index fd9f3ef..0000000
--- a/src/components/sponsors/sponsorSix.js
+++ /dev/null
@@ -1,30 +0,0 @@
-import React, {useState} from 'react'
-import './sponsors_body.css';
-
-const OVERLAY_STYLES = {
-    position: 'fixed',
-    top: 0,
-    left: 0,
-    right: 0,
-    bottom: 0,
-    backgroundColor: 'rgba(0, 0, 0, .7)',
-    zIndex: 1000
-}
-
-export default function SponsorSix({open, children, onClose}){
-    if (!open) return null
-    return (
-        <div style = {OVERLAY_STYLES}>
-            <div className="sponsorPopUp">
-                <button onClick={onClose}>&#x2715;</button>
-                <p className="Title">WC Branham</p>
-                <p>For over three decades, WC Branham has been a customer-focused family-owned 
-                business which excels in manufacturing power transmissions and fluid power systems. 
-                They are also a supplier of many other products that have served across the 
-                automotive, medical, and food processing industries.
-                </p>
-                <a href="https://www.wcbranham.com/index.cfm">https://www.wcbranham.com/</a>
-            </div>
-        </div>
-    )
-}
\ No newline at end of file
diff --git a/src/components/sponsors/sponsorSixteen.js b/src/components/sponsors/sponsorSixteen.js
deleted file mode 100644
index 5db73ee..0000000
--- a/src/components/sponsors/sponsorSixteen.js
+++ /dev/null
@@ -1,29 +0,0 @@
-import React, {useState} from 'react'
-import './sponsors_body.css';
-
-const OVERLAY_STYLES = {
-    position: 'fixed',
-    top: 0,
-    left: 0,
-    right: 0,
-    bottom: 0,
-    backgroundColor: 'rgba(0, 0, 0, .7)',
-    zIndex: 1000
-}
-
-export default function SponsorSixteen({open, children, onClose}){
-    if (!open) return null
-    return (
-        <div style = {OVERLAY_STYLES}>
-	        <div className="sponsorPopUp">
-				<button onClick={onClose}>&#x2715;</button>
-				<p className="Title">TCMetal Electric Steel</p>
-				<p>Provides high quality grain oriented and non-oriented electrical 
-                steel to the leading manufacturers in North America. With high 
-                efficiency cores that meet their high quality standards.
-                </p>
-                <a href="https://tcmetal.com/">https://tcmetal.com/</a>
-			</div>
-        </div>
-    )
-}
\ No newline at end of file
diff --git a/src/components/sponsors/sponsorTen.js b/src/components/sponsors/sponsorTen.js
deleted file mode 100644
index d94b7e6..0000000
--- a/src/components/sponsors/sponsorTen.js
+++ /dev/null
@@ -1,31 +0,0 @@
-import React, {useState} from 'react'
-import './sponsors_body.css';
-
-const OVERLAY_STYLES = {
-    position: 'fixed',
-    top: 0,
-    left: 0,
-    right: 0,
-    bottom: 0,
-    backgroundColor: 'rgba(0, 0, 0, .7)',
-    zIndex: 1000
-}
-
-export default function SponsorTen({open, children, onClose}){
-    if (!open) return null
-    return (
-        <div style = {OVERLAY_STYLES}>
-	        <div className="sponsorPopUp">
-				<button onClick={onClose}>&#x2715;</button>
-				<p className="Title">Titan Fittings</p>
-				<p>Stainless steel line is composed of over 10,000 stainless fittings 
-                in multiple product families that include hydraulic adapters, stainless 
-                hose ends, instrumental tube fittings, and Titan stainless valves. 
-                Company manufactures precise stainless components for large corporate and 
-                government clients nationally and has evolved into supplying high quality standard 
-                fittings for industrial applications globally. </p>
-                <a href="https://www.titanfittings.com/">https://www.titanfittings.com/</a>
-			</div>
-        </div>
-    )
-}
\ No newline at end of file
diff --git a/src/components/sponsors/sponsorThirteen.js b/src/components/sponsors/sponsorThirteen.js
deleted file mode 100644
index 42e18f7..0000000
--- a/src/components/sponsors/sponsorThirteen.js
+++ /dev/null
@@ -1,32 +0,0 @@
-import React, {useState} from 'react'
-import './sponsors_body.css';
-
-const OVERLAY_STYLES = {
-    position: 'fixed',
-    top: 0,
-    left: 0,
-    right: 0,
-    bottom: 0,
-    backgroundColor: 'rgba(0, 0, 0, .7)',
-    zIndex: 1000
-}
-
-export default function SponsorThirteen({open, children, onClose}){
-    if (!open) return null
-    return (
-        <div style = {OVERLAY_STYLES}>
-	        <div className="sponsorPopUp">
-				<button onClick={onClose}>&#x2715;</button>
-				<p className="Title">SICK Inc.</p>
-				<p>Manufactures sensor technology for anything from factory 
-                automation to logistics and process automation. These are 
-                sold and used in a wide range of industries, and create 
-                the perfect basis for controlling processes securely and 
-                efficiently, protecting individuals from accidents, and 
-                preventing damage to the environment.
-                </p>
-                <a href="https://www.sick.com/us/en/">https://www.sick.com/us/en/</a>
-			</div>
-        </div>
-    )
-}
\ No newline at end of file
diff --git a/src/components/sponsors/sponsorThree.js b/src/components/sponsors/sponsorThree.js
deleted file mode 100644
index 43f1ba2..0000000
--- a/src/components/sponsors/sponsorThree.js
+++ /dev/null
@@ -1,27 +0,0 @@
-import React, {useState} from 'react'
-import './sponsors_body.css';
-
-const OVERLAY_STYLES = {
-    position: 'fixed',
-    top: 0,
-    left: 0,
-    right: 0,
-    bottom: 0,
-    backgroundColor: 'rgba(0, 0, 0, .7)',
-    zIndex: 1000
-}
-
-export default function SponsorThree({open, children, onClose}){
-    if (!open) return null
-    return (
-        <div style = {OVERLAY_STYLES}>
-	        <div className="sponsorPopUp">
-				<button onClick={onClose}>&#x2715;</button>
-				<p className="Title">ANSYS</p>
-				<p>Develops and markets multiphysics engineering simulation software for product design, 
-                testing and operation and offers its products and services to customers worldwide.</p>
-                <a href="https://www.ansys.com/">https://www.ansys.com/</a>
-			</div>
-        </div>
-    )
-}
\ No newline at end of file
diff --git a/src/components/sponsors/sponsorTwelve.js b/src/components/sponsors/sponsorTwelve.js
deleted file mode 100644
index 2b22ef4..0000000
--- a/src/components/sponsors/sponsorTwelve.js
+++ /dev/null
@@ -1,29 +0,0 @@
-import React, {useState} from 'react'
-import './sponsors_body.css';
-
-const OVERLAY_STYLES = {
-    position: 'fixed',
-    top: 0,
-    left: 0,
-    right: 0,
-    bottom: 0,
-    backgroundColor: 'rgba(0, 0, 0, .7)',
-    zIndex: 1000
-}
-
-export default function SponsorTwelve({open, children, onClose}){
-    if (!open) return null
-    return (
-        <div style = {OVERLAY_STYLES}>
-	        <div className="sponsorPopUp">
-				<button onClick={onClose}>&#x2715;</button>
-				<p className="Title">Gerber Labs</p>
-				<p>The leading provider of printed circuit boards (PCB’s) for anyone 
-                from large companies to individuals working on home projects. Provides 
-                low-cost and high-quality custom boards made easily accessible to the public.
-                </p>
-                <a href="https://www.gerberlabs.com/">https://www.gerberlabs.com/</a>
-			</div>
-        </div>
-    )
-}
\ No newline at end of file
diff --git a/src/components/sponsors/sponsorTwenty.js b/src/components/sponsors/sponsorTwenty.js
deleted file mode 100644
index 2555478..0000000
--- a/src/components/sponsors/sponsorTwenty.js
+++ /dev/null
@@ -1,29 +0,0 @@
-import React, {useState} from 'react'
-import './sponsors_body.css';
-
-const OVERLAY_STYLES = {
-    position: 'fixed',
-    top: 0,
-    left: 0,
-    right: 0,
-    bottom: 0,
-    backgroundColor: 'rgba(0, 0, 0, .7)',
-    zIndex: 1000
-}
-
-export default function SponsorTwenty({open, children, onClose}){
-    if (!open) return null
-    return (
-        <div style = {OVERLAY_STYLES}>
-	        <div className="sponsorPopUp">
-				<button onClick={onClose}>&#x2715;</button>
-				<p className="Title">Composite One</p>
-				<p>Lead supplier in composite materials with high 
-                performance ratings and used across many industries 
-                including aerospace, construction and infrastructure.
-                </p>
-                <a href="https://www.compositesone.com/">https://www.compositesone.com/</a>
-			</div>
-        </div>
-    )
-}
\ No newline at end of file
diff --git a/src/components/sponsors/sponsorTwo.js b/src/components/sponsors/sponsorTwo.js
deleted file mode 100644
index 4ffa8d8..0000000
--- a/src/components/sponsors/sponsorTwo.js
+++ /dev/null
@@ -1,28 +0,0 @@
-import React, {useState} from 'react'
-import './sponsors_body.css';
-
-const OVERLAY_STYLES = {
-    position: 'fixed',
-    top: 0,
-    left: 0,
-    right: 0,
-    bottom: 0,
-    backgroundColor: 'rgba(0, 0, 0, .7)',
-    zIndex: 1000
-}
-
-export default function SponsorTwo({open, children, onClose}){
-    if (!open) return null
-    return (
-        <div style = {OVERLAY_STYLES}>
-	        <div className="sponsorPopUp">
-				<button onClick={onClose}>&#x2715;</button>
-				<p className="Title">Beckhoff</p>
-				<p>Works on the development and implementation of PC-based control technology all over the globes. 
-                This work spans over multiple different industries and product lines, cemented by their global presence.
-                </p>
-                <a href="https://www.beckhoff.com/en-us/">https://www.beckhoff.com/</a>
-			</div>
-        </div>
-    )
-}
\ No newline at end of file
diff --git a/src/components/sponsors/sponsors_body.css b/src/components/sponsors/sponsors_body.css
index 2b18c67..6108dec 100644
--- a/src/components/sponsors/sponsors_body.css
+++ b/src/components/sponsors/sponsors_body.css
@@ -41,12 +41,30 @@
 	background-color: #fff;
 }
 
+.sponsorHover {
+	display: flex;
+	width: 40vw;
+	height: 40vw;
+	margin-left: 1.5vw;
+	margin-right: 1.5vw;
+	margin-bottom: 3vw;
+	border: 15px solid goldenrod;
+	align-items: center;
+	background-color: #fff;
+}
+
 .sponsor img{
 	width: 25vw;
 	height: auto;
 	margin: auto;
 }
 
+.sponsorHover img{
+	width: 25vw;
+	height: auto;
+	margin: auto;
+}
+
 .sponsor-row-top-half, .sponsor-row-bottom-half{
 	display: flex;
 	flex-direction: row;
@@ -195,6 +213,14 @@
 		margin-bottom: 2vw;
 	}
 
+	.sponsorHover {
+		width: 17vw;
+		height: 17vw;
+		margin-left: 1vw;
+		margin-right: 1vw;
+		margin-bottom: 2vw;
+	}
+
 	.sponsorPopUp a{
 		margin-left: 0px;
 	}
@@ -203,6 +229,10 @@
 		width: 12vw;
 	}
 
+	.sponsorHover img{
+		width: 12vw;
+	}
+
 	.sponsor-ratings{
 		flex-direction: row;
 	}
diff --git a/src/components/sponsors/sponsors_body.js b/src/components/sponsors/sponsors_body.js
index 94ee86e..18621e6 100644
--- a/src/components/sponsors/sponsors_body.js
+++ b/src/components/sponsors/sponsors_body.js
@@ -1,110 +1,9 @@
-import './sponsors_body.css';
-
 import React, { useEffect, useState } from 'react';
-import SponsorOne from './sponsorOne';
-import SponsorTwo from './sponsorTwo';
-import SponsorThree from './sponsorThree';
-import SponsorFour from './sponsorFour';
-import SponsorFive from './sponsorFive';
-import SponsorSix from './sponsorSix';
-import SponsorSeven from './sponsorSeven';
-import SponsorEight from './sponsorEight';
-import SponsorNine from './sponsorNine';
-import SponsorTen from './sponsorTen';
-import SponsorEleven from './sponsorEleven';
-import SponsorTwelve from './sponsorTwelve';
-import SponsorThirteen from './sponsorThirteen';
-import SponsorFourteen from './sponsorFourteen';
-import SponsorFifteen from './sponsorFifteen';
-import SponsorSixteen from './sponsorSixteen';
-import SponsorSeventeen from './sponsorSeventeen';
-import SponsorEighteen from './sponsorEighteen';
-import SponsorNineteen from './sponsorNineteen';
-import SponsorTwenty from './sponsorTwenty';
-
-
-const Sponsors_body = () =>{
-	const [showResults1, setShowResults1] = React.useState(false)
-  	const onClick1 = () => setShowResults1(true)
-	const [isOpen1,setIsOpen1] = useState(false)
-
-	const [showResults2, setShowResults2] = React.useState(false)
-  	const onClick2 = () => setShowResults2(true)
-	const [isOpen2,setIsOpen2] = useState(false)
-
-	const [showResults3, setShowResults3] = React.useState(false)
-  	const onClick3 = () => setShowResults3(true)
-	const [isOpen3,setIsOpen3] = useState(false)
-
-	const [showResults4, setShowResults4] = React.useState(false)
-  	const onClick4 = () => setShowResults4(true)
-	const [isOpen4,setIsOpen4] = useState(false)
-
-	const [showResults5, setShowResults5] = React.useState(false)
-  	const onClick5 = () => setShowResults5(true)
-	const [isOpen5,setIsOpen5] = useState(false)
-
-	const [showResults6, setShowResults6] = React.useState(false)
-  	const onClick6 = () => setShowResults6(true)
-	const [isOpen6,setIsOpen6] = useState(false)
-
-	const [showResults7, setShowResults7] = React.useState(false)
-  	const onClick7 = () => setShowResults7(true)
-	const [isOpen7,setIsOpen7] = useState(false)
-
-	const [showResults8, setShowResults8] = React.useState(false)
-  	const onClick8 = () => setShowResults8(true)
-	const [isOpen8,setIsOpen8] = useState(false)
-
-	const [showResults9, setShowResults9] = React.useState(false)
-  	const onClick9 = () => setShowResults9(true)
-	const [isOpen9,setIsOpen9] = useState(false)
-
-	const [showResults10, setShowResults10] = React.useState(false)
-  	const onClick10 = () => setShowResults10(true)
-	const [isOpen10,setIsOpen10] = useState(false)
-
-	const [showResults11, setShowResults11] = React.useState(false)
-  	const onClick11 = () => setShowResults11(true)
-	const [isOpen11,setIsOpen11] = useState(false)
-
-	const [showResults12, setShowResults12] = React.useState(false)
-  	const onClick12 = () => setShowResults12(true)
-	const [isOpen12,setIsOpen12] = useState(false)
-
-	const [showResults13, setShowResults13] = React.useState(false)
-  	const onClick13 = () => setShowResults13(true)
-	const [isOpen13,setIsOpen13] = useState(false)
-
-	const [showResults14, setShowResults14] = React.useState(false)
-  	const onClick14 = () => setShowResults14(true)
-	const [isOpen14,setIsOpen14] = useState(false)
-
-	const [showResults15, setShowResults15] = React.useState(false)
-  	const onClick15 = () => setShowResults15(true)
-	const [isOpen15,setIsOpen15] = useState(false)
-
-	const [showResults16, setShowResults16] = React.useState(false)
-  	const onClick16 = () => setShowResults16(true)
-	const [isOpen16,setIsOpen16] = useState(false)
-
-	const [showResults17, setShowResults17] = React.useState(false)
-  	const onClick17 = () => setShowResults17(true)
-	const [isOpen17,setIsOpen17] = useState(false)
-
-	const [showResults18, setShowResults18] = React.useState(false)
-  	const onClick18 = () => setShowResults18(true)
-	const [isOpen18,setIsOpen18] = useState(false)
-
-	const [showResults19, setShowResults19] = React.useState(false)
-  	const onClick19 = () => setShowResults19(true)
-	const [isOpen19,setIsOpen19] = useState(false)
-
-	const [showResults20, setShowResults20] = React.useState(false)
-  	const onClick20 = () => setShowResults20(true)
-	const [isOpen20,setIsOpen20] = useState(false)
+import SponsorModal from "./sponsor";
+import './sponsors_body.css';
 
-	return(
+const Sponsors_body = () => {
+	return (
 		<React.Fragment>
 			<div className="oneloop-sponsors-body">
 				<div className="oneloop-sponsors-title">
@@ -182,72 +81,152 @@ const Sponsors_body = () =>{
 				<div className="sponsor-logos">
 					<div className="sponsor-row">
 						<div className="sponsor-row-top-half">
-							<button onClick={() => setIsOpen1(true)} className="sponsor"><img src="images/logo/etel-logo.png"/></button>
-							<SponsorOne open ={isOpen1} onClose = {() => setIsOpen1(false)}></SponsorOne>
-							<button onClick={() => setIsOpen2(true)} className="sponsor"><img src="images/logo/Beckhoff_Logo.svg.png"/></button>
-							<SponsorTwo open ={isOpen2} onClose = {() => setIsOpen2(false)}></SponsorTwo>
+							<SponsorModal
+								img={"images/logo/etel-logo.png"}
+								title={"ETEL"}
+								description={"ETEL is a company based in Switzerland which designs and builds motion systems in-house. They create systems that meet a range of needs from high-precision to high-power."}
+								url={"https://www.etel.ch"}
+							/>
+							<SponsorModal
+								img={"images/logo/Beckhoff_Logo.svg.png"}
+								title={"Beckhoff"}
+								description={"Works on the development and implementation of PC-based control technology all over the globes. This work spans over multiple different industries and product lines, cemented by their global presence."}
+								url={"https://www.beckhoff.com/en-us/"}
+							/>
 						</div>
 						<div className="sponsor-row-bottom-half">
-							<button onClick={() => setIsOpen3(true)} className="sponsor"><img src="images/logo/ANSYS_Logo.png"/></button>
-							<SponsorThree open ={isOpen3} onClose = {() => setIsOpen3(false)}></SponsorThree>
-							<button onClick={() => setIsOpen4(true)} className="sponsor"><img src="images/logo/Valispace_Logo.png"/></button>
-							<SponsorFour open ={isOpen4} onClose = {() => setIsOpen4(false)}></SponsorFour>
+							<SponsorModal
+								img={"images/logo/ANSYS_Logo.png"}
+								title={"ANSYS"}
+								description={"Develops and markets multiphysics engineering simulation software for product design, testing and operation and offers its products and services to customers worldwide."}
+								url={"https://www.ansys.com"}
+							/>
+							<SponsorModal
+								img={"images/logo/Valispace_Logo.png"}
+								title={"Valispace"}
+								description={"Helps engineers streamline their engineering efforts by centralizing ‘loose’ engineering data into one place. This provides a way for engineers to maximize their time and effort and design with their system of detailed documentation and secured user deployment."}
+								url={"https://www.valispace.com/"}
+							/>
 						</div>
 					</div>
 					<div className="sponsor-row">
 						<div className="sponsor-row-top-half">
-							<button onClick={() => setIsOpen5(true)} className="sponsor"><img src="images/logo/siemens-logo.png"/></button>
-							<SponsorFive open ={isOpen5} onClose = {() => setIsOpen5(false)}></SponsorFive>
-							<button onClick={() => setIsOpen6(true)} className="sponsor"><img src="images/logo/WCBranham_logo.gif"/></button>
-							<SponsorSix open ={isOpen6} onClose = {() => setIsOpen6(false)}></SponsorSix>
+							<SponsorModal
+								img={"images/logo/siemens-logo.png"}
+								title={"Seimens"}
+								description={"Siemens is the largest manufacturing company in Europe. A multinational conglomerate company whose varied acquisitions allowed it to be involved in wind power, gas power, energy management, transportation, health care, automation and more."}
+								url={"https://www.siemens.com/global/en.html"}
+							/>
+							<SponsorModal
+								img={"images/logo/WCBranham_logo.gif"}
+								title={"WC Branham"}
+								description={"For over three decades, WC Branham has been a customer-focused family-owned business which excels in manufacturing power transmissions and fluid power systems. They are also a supplier of many other products that have served across the automotive, medical, and food processing industries."}
+								url={"https://www.wcbranham.com/index.cfm"}
+							/>
 						</div>
 						<div className="sponsor-row-bottom-half">
-							<button onClick={() => setIsOpen7(true)} className="sponsor"><img src="images/logo/air-caster-squarelogo.png"/></button>
-							<SponsorSeven open ={isOpen7} onClose = {() => setIsOpen7(false)}></SponsorSeven>
-							<button onClick={() => setIsOpen8(true)} className="sponsor"><img src="images/logo/FluidFlow_logo.jpg"/></button>
-							<SponsorEight open ={isOpen8} onClose = {() => setIsOpen8(false)}></SponsorEight>
+							<SponsorModal
+								img={"images/logo/air-caster-squarelogo.png"}
+								title={"AirCaster"}
+								description={"AirCaster designs & builds air bearing systems that transport heavy loads across industries which includes but is not limited to aerospace, oil, defense, and automotive. They are known for their durability and customization to meet unique needs."}
+								url={"https://www.aircaster.com"}
+							/>
+							<SponsorModal
+								img={"images/logo/FluidFlow_logo.jpg"}
+								title={"Fluid Flow"}
+								description={"FluidFlow is a pipe flow design software allowing users to design pressure drop systems of any needs. Flows can include non-newtonian and slurry flow, two-phase flow, thermal energy transfer and compressible flow for example."}
+								url={"https://fluidflowinfo.com"}
+							/>
 						</div>
 					</div>
 					<div className="sponsor-row">
 						<div className="sponsor-row-top-half">
-							<button onClick={() => setIsOpen9(true)} className="sponsor"><img src="images/logo/AquaEnvironment_Logo.png"/></button>
-							<SponsorNine open ={isOpen9} onClose = {() => setIsOpen9(false)}></SponsorNine>
-							<button onClick={() => setIsOpen10(true)} className="sponsor"><img src="images/logo/TitanFittings_logo.png"/></button>
-							<SponsorTen open ={isOpen10} onClose = {() => setIsOpen10(false)}></SponsorTen>
+							<SponsorModal
+								img={"images/logo/AquaEnvironment_Logo.png"}
+								title={"Aqua Environment"}
+								description={"Specialize in manufacturing high pressure components such as valves and regulators for gas and fluid control. Products are used in wide rage of commercial firefighting, military, sports-diving, natural gas, natural gas fill stations, and other applications (shipping worldwide)."}
+								url={"https://www.aquaenvironmentinc.com/index.html"}
+							/>
+							<SponsorModal
+								img={"images/logo/TitanFittings_logo.png"}
+								title={"Titan Fittings"}
+								description={"Stainless steel line is composed of over 10,000 stainless fittings in multiple product families that include hydraulic adapters, stainless hose ends, instrumental tube fittings, and Titan stainless valves. Company manufactures precise stainless components for large corporate and government clients nationally and has evolved into supplying high quality standard fittings for industrial applications globally."}
+								url={"https://www.titanfittings.com/"}
+							/>
 						</div>
 						<div className="sponsor-row-bottom-half">
-							<button onClick={() => setIsOpen11(true)} className="sponsor"><img src="images/logo/uc-davis-mechanical-aerospace-engineering-new-faculty-2020.jpg"/></button>
-							<SponsorEleven open ={isOpen11} onClose = {() => setIsOpen11(false)}></SponsorEleven>
-							<button onClick={() => setIsOpen12(true)} className="sponsor"><img src="images/logo/Gerberlabs_logo.png"/></button>
-							<SponsorTwelve open ={isOpen12} onClose = {() => setIsOpen12(false)}></SponsorTwelve>
+							<SponsorModal
+								img={"images/logo/uc-davis-mechanical-aerospace-engineering-new-faculty-2020.jpg"}
+								title={"Mechanical and Aerospace Engineering Department"}
+								description={"Department offering comprehensive resources for students, faculty, and staff at UC Davis (offering undergraduate advising and research opportunities)."}
+								url={"https://mae.ucdavis.edu"}
+							/>
+							<SponsorModal
+								img={"images/logo/Gerberlabs_logo.png"}
+								title={"Gerber Labs"}
+								description={"The leading provider of printed circuit boards (PCB’s) for anyone from large companies to individuals working on home projects. Provides low-cost and high-quality custom boards made easily accessible to the public."}
+								url={"https://www.gerberlabs.com/"}
+							/>
 						</div>
 					</div>
 					<div className="sponsor-row">
 						<div className="sponsor-row-top-half">
-							<button onClick={() => setIsOpen13(true)} className="sponsor"><img src="images/logo/SICKInc_Logo.jpg"/></button>
-							<SponsorThirteen open ={isOpen13} onClose = {() => setIsOpen13(false)}></SponsorThirteen>
-							<button onClick={() => setIsOpen14(true)} className="sponsor"><img src="images/logo/Rockwell_Automation_Logo.png"/></button>
-							<SponsorFourteen open ={isOpen14} onClose = {() => setIsOpen14(false)}></SponsorFourteen>
+							<SponsorModal
+								img={"images/logo/SICKInc_Logo.jpg"}
+								title={"SICK Inc."}
+								description={"Manufactures sensor technology for anything from factory automation to logistics and process automation. These are sold and used in a wide range of industries, and create the perfect basis for controlling processes securely and efficiently, protecting individuals from accidents, and preventing damage to the environment."}
+								url={"https://www.sick.com/us/en/"}
+							/>
+							<SponsorModal
+								img={"images/logo/Rockwell_Automation_Logo.png"}
+								title={"Rockwell Automation"}
+								description={"A provider of both information technology and industrial automation. Many of their products are recognized across the globe for excellence."}
+								url={"https://www.rockwellautomation.com/en-us.html"}
+							/>
 						</div>
 						<div className="sponsor-row-bottom-half">
-							<button onClick={() => setIsOpen15(true)} className="sponsor"><img src="images/logo/Citris_logo.svg"/></button>
-							<SponsorFifteen open ={isOpen15} onClose = {() => setIsOpen15(false)}></SponsorFifteen>
-							<button onClick={() => setIsOpen16(true)} className="sponsor"><img src="images/logo/TCMetal_Electric_Steel_Logo.png"/></button>
-							<SponsorSixteen open ={isOpen16} onClose = {() => setIsOpen16(false)}></SponsorSixteen>
+							<SponsorModal
+								img={"images/logo/Citris_logo.svg"}
+								title={"CITRIS"}
+								description={"An institute opened by the university of california to help further the research might of the general university of california system, and help it to connect with the entrepreneurial spirit of silicon valley, bridging the gap between companies, students, and policy makers. "}
+								url={"https://citris-uc.org"}
+							/>
+							<SponsorModal
+								img={"images/logo/TCMetal_Electric_Steel_Logo.png"}
+								title={"TCMetal Electric Steel"}
+								description={"Provides high quality grain oriented and non-oriented electrical steel to the leading manufacturers in North America. With high efficiency cores that meet their high quality standards."}
+								url={"https://tcmetal.com/"}
+							/>
 						</div>
 					</div>
 					<div className="sponsor-row">
 						<div className="sponsor-row-top-half">
-							<button onClick={() => setIsOpen17(true)} className="sponsor"><img src="images/logo/KeySight_Technology_Logo.png"/></button>
-							<SponsorSeventeen open ={isOpen17} onClose = {() => setIsOpen17(false)}></SponsorSeventeen>
-							<button onClick={() => setIsOpen18(true)} className="sponsor"><img src="images/logo/SecondOrderEffects_Logo.png"/></button>
-							<SponsorEighteen open ={isOpen18} onClose = {() => setIsOpen18(false)}></SponsorEighteen>
+							<SponsorModal
+								img={"images/logo/KeySight_Technology_Logo.png"}
+								title={"Keysight Technology"}
+								description={"Provides many services in technology consultation, design, and software to address pressing issues for business and technological advancement."}
+								url={"https://www.keysight.com/us/en/home.html"}
+							/>
+							<SponsorModal
+								img={"images/logo/SecondOrderEffects_Logo.png"}
+								title={"Second Order Effects"}
+								description={"Develops and designs many solutions to challenges that are solved by their broad range of expertise in electronics, automotive, healthcare, and industrial automation industries. Reaching a solution that produces maximized results."}
+								url={"https://soeffects.com"}
+							/>
 						</div>
 						<div className="sponsor-row-bottom-half">
-							<button onClick={() => setIsOpen19(true)} className="sponsor"><img src="images/logo/Ubiquiti_Networks_Logo.svg.png"/></button>
-							<SponsorNineteen open ={isOpen19} onClose = {() => setIsOpen19(false)}></SponsorNineteen>
-							<button onClick={() => setIsOpen20(true)} className="sponsor"><img src="images/logo/CompositeOne_Logo.png"/></button>
-							<SponsorTwenty open ={isOpen20} onClose = {() => setIsOpen20(false)}></SponsorTwenty>
+							<SponsorModal
+								img={"images/logo/Ubiquiti_Networks_Logo.svg.png"}
+								title={"Ubiquiti Networks"}
+								description={"A provider of multi-application network operating systems that offers many services such as identification across networks, network management, and IT customization."}
+								url={"https://www.ui.com"}
+							/>
+							<SponsorModal
+								img={"images/logo/CompositeOne_Logo.png"}
+								title={"Composite One"}
+								description={"Lead supplier in composite materials with high performance ratings and used across many industries including aerospace, construction and infrastructure."}
+								url={"https://www.compositesone.com"}
+							/>
 						</div>
 					</div>
 				</div>
diff --git a/src/components/team/team2020_body.js b/src/components/team/team2020_body.js
index 7bd5718..4b9f015 100644
--- a/src/components/team/team2020_body.js
+++ b/src/components/team/team2020_body.js
@@ -404,12 +404,12 @@ const Team2020_body = () =>{
 							<span>Stability Lead</span></h4>
 						</div>
 						<div className="team-member">
-							<img className="team-member" src="/images/team/20202021/Mark_Hagiwara.jpeg"/>
+							<img className="team-member" src="/images/team/20202021/Anjali_Gupta.jpeg"/>
 							<h4>ANJALI GUPTA<br/>
 							<span>Stability Co-Lead</span></h4>
 						</div>
 						<div className="team-member">
-							<img className="team-member" src="/images/team/20202021/Mark_Hagiwara.jpeg"/>
+							<img className="team-member" src="/images/team/20202021/Kansas_Seung.jpeg"/>
 							<h4>KANSAS SEUNG<br/>
 							<span>Stability Co-Lead</span></h4>
 						</div>
@@ -421,19 +421,19 @@ const Team2020_body = () =>{
 							<span>Stability Member</span></h4>
 						</div>
 						<div className="team-member">
-							<img className="team-member" src="/images/team/20202021/Kyle_Heinzman.jpeg"/>
+							<img className="team-member" src="/images/team/20202021/Andrew_Gunlock.jpeg"/>
 							<h4>ANDREW GUNLOCK<br/>
 							<span>Stability Member</span></h4>
 						</div>
 						<div className="team-member">
-							<img className="team-member" src="/images/team/20202021/Kyle_Heinzman.jpeg"/>
+							<img className="team-member" src="/images/team/20202021/Weijin_Tang.jpeg"/>
 							<h4>WEIJIN TANG<br/>
 							<span>Stability Member</span></h4>
 						</div>
 					</div>
 					<div className="stability-row row row-3">
 						<div className="team-member">
-							<img className="team-member" src="/images/team/20202021/Kyle_Heinzman.jpeg"/>
+							<img className="team-member" src="/images/team/20202021/Zoe_Wilf.jpeg"/>
 							<h4>ZOE WILF<br/>
 							<span>Stability Member</span></h4>
 						</div>
